### ISSUE TITLE
Phish domain / management domain?

### ISSUE THREAD

**Issue by jameskf** [pentestgeek/phishing-frenzy#157]
Hi there,

I have successfully set up Phishing Frenzy on a Kali box. It works well except for one thing, and it might be that I am trying to do something incorrectly.

My intention was to have one domain set up with HTTPS, which is used to access the Phishing Frenzy management site, and another domain or domains running over regular HTTP which are my phishing pages.

This all seemed to work well - I was able to create the phishing campaign, send the emails, the link in the email has the unique ID, and the phishing page is served up nicely from the phishing domain. The issue is that I am not getting any click stats when the phishing page is accessed.

I had a look at the troubleshooting page and did a couple of tests:

"http://phishing-domain.com/reports/results/?uid=xxxxxxxxxx" generates a 404 error, but
"https://management-domain.com/reports/results/?uid=xxxxxxxxxx" returns "start, uid check, length 1, over 1," which in turn increments the click stats on my campaign.

Seems to me that I need to run the whole box off a single domain for the stats to work. But if that was so, why would it be necessary to specify the FQDN in the campaign details? The box already knows its identity from the application.rb file.

Anyway, I figure I've messed up somewhere and would be grateful for any assistance you can provide!

Cheers
James

**Comment by zeknox** [comment]
This should work the way you want it too. If you are only having issues with the tracking clicks you've gotta have an issue with SITE_URL within application.rb.  It should be configured with something like:

<pre>
SITE_URL = "https://management-domain.com"
</pre>

This will define where the management interface is for all the phishing sites so you can run your phishing site on any domain other than management-domain.com. PF will automatically add PHP tags to the phishing site which is used to call back to the PF interface based on whats defined as SITE_URL.

SITE_URL is almost always going to be the FQDN of how your accessing the PF interface.

**Comment by jameskf** [comment]
Thanks for the info, I got it fixed - looks like a combination of a couple of issues. I should mention at this point that this box is sitting up in AWS.

1. I had my SITE_URL set to `http://phishing-domain.com`. I changed it to `https://management-domain.com`, performed a `apachectl restart` and deactivated/reactivated the campaign but was still having the same issue.

2. When I loaded the page after making the above changes, I noticed that the page took an unusually long time to load. There was no unusual CPU activity but a `netstat` showed a SYN_SENT from my internal IP to the IP that `management-domain.com` resolves to. When I tried to manually access `management-domain.com` from the command line this behaviour was confirmed (i.e. it didn't work and just hung for ages).

The fix (in addition to fixing my SITE_URL variable) was to add a host file entry so that `management-domain.com` resolves to my internal AWS IP address instead of my external AWS IP address.

After this I was able to hit `management-domain.com` from the command line without issue, and I suppose this let all the php stuff in the background work properly also.

All up and running now! Thanks heaps for your speedy response.
